.github/workflows/catkin.yml

```diff
@@ -0,0 +1,26 @@
+name: Catkin Build and Test
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_CMD: >-
+        . /opt/ros/noetic/setup.zsh &&
+        cd catkin_ws &&
+        catkin build --no-status --force-color &&
+        catkin test --no-status --force-color
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: src/rocket_league
+    - name: Build the local Docker image
+      run: ./src/rocket_league/docker/docker-build.sh --build-arg TAG=${{ github.event.number }}
```

Comment:
We only want to use a non-default tag if the PR forced a new Docker image to be built. And if that was the case, this action needs to wait until the other action finishes.